kolibri/utils/sanity_checks.py

```diff
@@ -93,4 +94,4 @@ def check_log_file_location():
         old_log_path = os.path.join(home, log)
         if os.path.exists(old_log_path):
             new_log_path = os.path.join(home, "logs", log_location_update[log])
-            os.rename(old_log_path, new_log_path)
+            shutil.move(old_log_path, new_log_path)
```

Comment:
Ah yes, that will make the difference!

Wonder if we should also just wrap this in a try except clause, so that we don't break startup if the file is corrupted, is actually a broken symlink etc.?

Reply:
I think `shutil.move` doesn't find out source files with broken symlinks. I created a corrupted file and used `shutil.move` to rename it to something else, but no errors showed up.

I have the `os.path.exists` check here: https://github.com/learningequality/kolibri/pull/5609/files/49ee1a89bbb51e9f2500b98883a1f197fb5cb9b9#diff-d2fc08a4eefe48248e5c777e07b056fdR95
so if the source file is corrupted, it will be ignored.